build.sh

```diff
@@ -11,5 +11,5 @@ docs/check.sh
 runInBuilder "set -e;
     ./gradlew help;
     ./gradlew help -PuseCompositeBuild=false;
-    ./gradlew -p subprojects build ${GRADLE_ARGS};
+    ./gradlew -p subprojects build --scan ${GRADLE_ARGS};
```

Comment:
It requires to accept a license agreement from stdin.
We can avoid this by applying Gradle enterprise plugin.

Reply:
#473